lib/src/config/mod.rs

```diff
@@ -35,7 +35,7 @@
 //!     listen on
 //!     * examples: `"localhost"`, `"0.0.0.0"`, `"1.2.3.4"`
 //!   * **port**: _[integer]_ a port number to listen on
-//!     * examples: `"8000"`, `"80"`, `"4242"`
+//!     * examples: `"8000"`, `"54321"`, `"4242"`
```

Comment:
You can leave the `80` here and change the `8000`.

Reply:
Done